Release 1.9.1 (#1855)
hedgedoc/hedgedoc: public/docs/release-notes.md

# Release Notes

## <i class="fa fa-tag"></i> 1.9.1 <i class="fa fa-calendar-o"></i> 2021-12-02

> Starting with this release HedgeDoc will support only the LTS versions of node.
> Please make sure that you use the latest minor and patch version.

Reviewer: ```suggestion
This release increases the minimum required Node versions to `12.20.0`, `14.13.1` and `16`.
In general, only the latest releases of Node 12, 14 and 16 are officially supported by us, older minor versions can be dropped at any time.
We recommend you run HedgeDoc with the latest release of Node 16.
```

### Bugfixes
- Add workaround for incorrect CSP handling in Safari

> - Fix crash when response from GitLab API is not ok.

Reviewer: ```suggestion
- Fix crash when an unexpected response from the GitLab API is encountered
```
